document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/AccountsDocumentInfoServiceImpl.java

```diff
@@ -1,12 +1,36 @@
 package uk.gov.companieshouse.document.generator.accounts;
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.api.model.transaction.Transaction;
+import uk.gov.companieshouse.document.generator.accounts.service.TransactionService;
 import uk.gov.companieshouse.document.generator.interfaces.DocumentInfoService;
 import uk.gov.companieshouse.document.generator.interfaces.model.DocumentInfo;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
 
+@Component
 public class AccountsDocumentInfoServiceImpl implements DocumentInfoService {
 
+    @Autowired
+    private TransactionService transactionService;
+
+    public static final String MODULE_NAME_SPACE = "document-generator-accounts";
```

Comment:
Rightly or wrongly, we're storing application namespace in the main files now -see https://github.com/companieshouse/company-accounts.api.ch.gov.uk/blob/develop/src/main/java/uk/gov/companieshouse/api/accounts/CompanyAccountsApplication.java#L22

Reply:
Very true, I believe it is the correct thing to do as it is all the same in most cases, just saves someone having to change the name in multiple places. We have also done it for `document-generator-core` however since we are using maven module, it would make sense that we split it to module specific logging as you don't want a `document-generator-accounts` log to come out under `document-generator-core`. Additionally `document-generator-accounts` hasn't got a main file in regards to `SpringBoot` and `main` but this is the only entry point to this module so it made sense to put it in here.

Reply:
Fair point, disregard my comment!